README.md

```diff
@@ -29,31 +29,26 @@ public void ConfigureServices(IServiceCollection services)
 {
     services.AddSingleton<ChatSchema>();
 
-    // add http transport    
-    services.AddGraphQLHttp();
-    
-    // setup execution options for ChatSchema
-    services.Configure<ExecutionOptions<ChatSchema>>(options =>
-            {
-                options.EnableMetrics = true;
-                options.ExposeExceptions = true;
-                options.UserContext = "something";
-            });
-
-    // add websocket transport for ChatSchema
-    services.AddGraphQLWebSocket<ChatSchema>();
+    // Add GraphQL services and configure options
+    services.AddGraphQL(options =>
+    {
+        options.EnableMetrics = true;
+        options.ExposeExceptions = this.Environment.IsDevelopment();
+    })
+    .AddWebSockets() // Add required services for web socket support
+    .AddDataLoader(); // Add required services for DataLoader support
 }
 
 public void Configure(IApplicationBuilder app, IHostingEnvironment env)
 {
     // this is required for websockets support
     app.UseWebSockets();
 
-    // use websocket middleware for ChatSchema at default url /graphql
-    app.UseGraphQLWebSocket<ChatSchema>(new GraphQLWebSocketsOptions());
+    // use websocket middleware for ChatSchema at path /graphql
+    app.UseGraphQLWebSockets<ChatSchema>("/graphql");
```

Comment:
I think it will be rare that someone would want to change the path. Can `/graphql` be defaulted for `UseGraphQLWebSockets` and `UseGraphQL`?